python/dgl/distributed/kvstore.py

```diff
@@ -60,7 +60,7 @@ def process_request(self, server_state):
         if kv_store.data_store.__contains__(self.name) is False:
             raise RuntimeError("KVServer Cannot find data tensor with name: %s" % self.name)
```

Comment:
Should we add a check: `kv_store.pull_handlers.__contains__(self.name) is False:`

Reply:
whenever a data is initialized, a pull and push handler is registered. i think we don't need to check.